bundle/config/bundle.go

```diff
@@ -26,10 +26,10 @@ type Bundle struct {
 	Terraform *Terraform `json:"terraform,omitempty" bundle:"readonly"`
 
 	// Lock configures locking behavior on deployment.
-	Lock Lock `json:"lock" bundle:"readonly"`
+	Lock Lock `json:"lock,omitempty" bundle:"readonly"`
```

Comment:
I don't think Lock can be ever empty

Reply:
Done. It's a no-op anyways because the "omitempty" tag does not apply to struct fields. Leftover from a previous attempt to make this `*Lock` instead which turns out is not worth the trouble.